modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerAlterFilterTest.java

```diff
@@ -175,54 +164,64 @@ void testAlterFilterForDefaultZone() throws Exception {
      *
      * @throws Exception If failed.
      */
-    @Test
-    void testAlterFilterDoNotAffectScaleDown() throws Exception {
-        preparePrerequisites(IMMEDIATE_TIMER_VALUE, 10000);
+    @ParameterizedTest
+    @ValueSource(booleans =  {true, false})
```

Comment:
What about using zone itself as a parameter instead of boolean?

Reply:
done, thanks!